### ISSUE TITLE
oscapd-cli help should provide list of possible actions

### ISSUE THREAD

**Issue by jan-cerny** [OpenSCAP/openscap-daemon#52]
I have recent daemon from git on Fedora 23, Python3. 

I think that when I want to show help for a particular module, it should give me a list of possible actions. But now it shows nothing.

```
[root@localhost ~]# oscapd-cli task -h
usage: oscapd-cli task [-h] [-f] [-r]
                       [TASK_ID] [ACTION] [actions [actions ...]]

positional arguments:
  TASK_ID               ID of the task to display, if none is provided a
                        summary of all tasks is displayed
  ACTION
  actions               parameters

optional arguments:
  -h, --help            show this help message and exit
  -f, --force           remove task without confirmation
  -r, --remove-results  remove with results
```

I would expect there possible values of the ACTION argument.

**Comment by mpreisler** [comment]
Yeah, you are right. We should have done the task module using subparsers, similar to the root module. Instead we are using a nasty hack :-/ I will see if this can be fixed in a way that doesn't break CLI.

**Comment by mpreisler** [comment]
Fixed in 80640dd287c62685c2b4f7ee04a6f836e2eb4601

```
$ python3 ./oscapd-cli task 1 -h
usage: oscapd-cli task [-h] [-f] [-r]
                       [TASK_ID] [ACTION] [parameter [parameter ...]]

positional arguments:
  TASK_ID               ID of the task to display, or perform action on. If
                        none is provided a summary of all tasks is displayed.
  ACTION                Which action to perform on selected task. Use one of
                        info, guide, run, enable, disable, remove, set-
                        schedule-repeat-after, set-online-remediation, set-
                        schedule-not-before, set-target, set-profile-id, set-
                        input, set-title, set-enabled, set-tailoring
  parameter             Parameters for the ACTION. For setter actions this is
                        the string that you want to set. Some actions, such as
                        enable, remove, ... don't require any parameters.

optional arguments:
  -h, --help            show this help message and exit
  -f, --force           remove task without confirmation
  -r, --remove-results  remove with results
```

```
$ python2 ./oscapd-cli task 1 -h
usage: oscapd-cli task [-h] [-f] [-r]
                       [TASK_ID] [ACTION] [parameter [parameter ...]]

positional arguments:
  TASK_ID               ID of the task to display, or perform action on. If
                        none is provided a summary of all tasks is displayed.
  ACTION                Which action to perform on selected task. Use one of
                        info, guide, run, enable, disable, remove, set-target,
                        set-schedule-repeat-after, set-online-remediation,
                        set-profile-id, set-title, set-enabled, set-schedule-
                        not-before, set-tailoring, set-input
  parameter             Parameters for the ACTION. For setter actions this is
                        the string that you want to set. Some actions, such as
                        enable, remove, ... don't require any parameters.

optional arguments:
  -h, --help            show this help message and exit
  -f, --force           remove task without confirmation
  -r, --remove-results  remove with results
```
